tests/manual_jobs/test_job_statuses.py

```diff
@@ -0,0 +1,27 @@
+import pytest
```

Comment:
Those are already in `test_job.py`. I was thinking of moving them to `test_job_statuses.py` in a subsequent PR, since it falls outside the scope of async jobs